cmd/mirror.go

```diff
@@ -221,80 +118,192 @@ func newMirrorSetCmd() *cobra.Command {
 	return cmd
 }
 
+// the `mirror introduce` sub command
+func newMirrorIntroduceCmd() *cobra.Command {
+	name := ""
+	privPath := ""
+
+	cmd := &cobra.Command{
+		Use:   "introduce <id>",
+		Short: "introduce a new owner",
+		Long:  "introduce a new owner to current mirror",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 1 {
+				return cmd.Help()
+			}
+
+			id := args[0]
+			if name == "" {
+				name = id
+			}
+
+			// the privPath can point to a public key becase the Public method of KeyInfo works on both priv and pub key
+			privKey, err := loadPrivKey(privPath)
+			if err != nil {
+				return err
+			}
+			pubKey, err := privKey.Public()
+			if err != nil {
+				return err
+			}
+
+			env := environment.GlobalEnv()
+			return env.V1Repository().Mirror().Introduce(id, name, pubKey)
+		},
+	}
+
+	cmd.Flags().StringVarP(&name, "name", "n", "", "the name of the owner, default: id of the owner")
+	cmd.Flags().StringVarP(&privPath, "key", "k", "", "private/public key path of the owner")
```

Comment:
I think we should make the description more clear if only `private key` can be assigned.
`private/public key path of the owner` -> `Specify the private key path of the owner`.

IMO, all descriptions for flags should be started with a upper case.